qiskit/optimization/converters/ising_to_quadratic_program.py

```diff
@@ -173,3 +178,42 @@ def _create_qubo_matrix(self):
             x_index = [i for i, x in enumerate(lst_x) if x is True]
             if len(x_index) > 0:
                 raise QiskitOptimizationError('Pauli Xs exist in the Pauli {}'.format(pauli.x))
+
+    @property
+    def linear(self) -> bool:
+        r"""
+        Returns the conversion mode for quadratic terms
+
+        Returns:
+            If it's True :math:`x^2` is treated as a linear term.
+            Else, :math:`x^2` is treated as a quadratic term.
+        """
+        return self._linear
+
+    @linear.setter  # type:ignore
+    def linear(self, linear: bool) -> None:
+        """Set a conversion mode for quadratic terms
+
+        Args:
+            linear: If it's True :math:`x^2` is treated as a linear term.
+                Else, :math:`x^2` is treated as a quadratic term.
+        """
+        self._linear = linear
+
+    @property
+    def name(self) -> str:
+        """Returns the name of the converted problem
+
+        Returns:
+            The name of the converted problem
+        """
+        return self._name
+
+    @name.setter  # type:ignore
+    def name(self, name: Optional[str]) -> None:
```

Comment:
Should the name here be Optional? In the init it defaults to empty string ' ' and the getter here does not support return of Optional

Reply:
see later comment on `name`.